api/src/opentrons/protocol_reader/file_reader_writer.py

```diff
@@ -34,10 +35,14 @@ class FileReadError(Exception):
     """An error raised if input files cannot be read."""
 
 
+class UnknownJsonFileError(Exception):
```

Comment:
Might make sense to inherit from `FileReadError`

```suggestion
class UnknownJsonFileError(FileReadError):
```